python/activator/visit.py

```diff
@@ -1,24 +1,59 @@
 __all__ = ["Visit"]
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+import enum
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True, kw_only=True)
 class Visit:
-    # elements must be hashable and JSON-persistable; built-in types recommended
-    instrument: str    # short name
+    # Elements must be hashable and JSON-persistable; built-in types
+    # recommended. list is not hashable, but gets special treatment because
```

Comment:
Should list be capitalized? Not entirely sure in this case.

Reply:
I'd rather use the type's intrinsic capitalization; "List" could be confusing.

Reply:
That's what I thought, I just wasn't 100% sure.